engine-rest/engine-rest-openapi/src/main/templates/paths/history/detail/post.ftl

```diff
@@ -0,0 +1,152 @@
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "queryHistoricDetails"
+      tag = "Historic Detail"
+      summary = "Get Historic Details (POST)"
+      desc = "Queries for historic details that fulfill the given parameters. This method is slightly more
+              powerful than the [Get Historic Details](${docsUrl}/reference/rest/history/detail/get-detail-query/)
+              method because it allows sorting by multiple parameters. The size of the result set can be retrieved by
+              using the [Get Historic Detail Count](${docsUrl}/reference/rest/history/detail/get-detail-query-count/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#include "/lib/commons/pagination-params.ftl">
+    <#assign last = true >
+    <#include "/lib/commons/deserialize-values-parameter.ftl">
+
+  ],
+
+    <@lib.requestBody
+        mediaType = "application/json"
+        dto = "HistoricDetailQueryDto"
+        examples = ['"example-1": {
+                         "summary": "POST `/job/count`",
+                         "value": {
+                           "dueDates": [
+                             {
+                               "operator": "gt",
+                               "value": "2012-07-17T17:00:00.000+0200"
+                             },
+                             {
+                               "operator": "lt",
+                               "value": "2012-07-17T18:00:00.000+0200"
+                             }
+                           ],
+                           "createTimes": [
+                             {
+                               "operator": "gt",
+                               "value": "2012-05-05T10:00:00.000+0200"
+                             },
+                             {
+                               "operator": "lt",
+                               "value": "2012-07-16T15:00:00.000+0200"
+                             }
+                           ]
+                         }
+                       }']
+    />
+
+  "responses": {
+
+    <@lib.response
+        code = "200"
+        dto = "HistoricDetailDto"
+        array = true
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "POST `/history/detail?firstResult=1&maxResults=10`",
+                       "description": "A JSON object with the following properties:",
+                       "value": {
+                         "processInstanceId": "3cd597b7-001a-11e7-8c6b-34f39ab71d4e",
+                         "occurredAfter": "2018-01-29T10:15:45.000+0100",
+                         "sorting": [
+                           {
+                             "sortBy": "processInstanceId",
+                             "sortOrder": "asc"
+                           }
+                         ]
+                       }
+                     },
+                     "example-2":{
```

Comment:
Where does this example come from? If it's new should we add it in the docs as well?